# eQTL-workflow-parameter

## LncRNAseq-workflow-parameter
| lncRNA-seq | 数据库/软件 | 功能                  | 版本       | 参数                                                         | 参考文献/数据库地址                                          |
| ---------- | ----------- | --------------------- | ---------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
| lncRNA-seq | Rfam        | RNA功能注释和家族分析 | V13        | \                                                            | http://pfam.xfam.org/                                        |
| lncRNA-seq | miRBase     | lnc与已知前体比对     | Release 21 | \                                                            | http://www.mirbase.org/                                      |
| lncRNA-seq | fastp       | 低质量数据过滤        | v0.20.0    |                                                              | Chen, S., Zhou, Y., Chen, Y., & Gu, J. (2018). fastp: An ultra-fast  all-in-one FASTQ preprocessor. Bioinformatics, 34(17), i884–i890.  https://doi.org/10.1093/bioinformatics/bty560 |
| lncRNA-seq | bowtie2     | 核糖体比对            | v2.2.8     |                                                              | Langmead B, Salzberg S L. Fast gapped-read alignment with Bowtie 2[J].  Nature methods, 2012, 9(4): 357. |
| lncRNA-seq | HISAT2      | 基因组比对、可变剪接  | v2.1.1     | #NAME?                                                       | Kim, Daehwan, Ben Langmead, and  Steven L. Salzberg. "HISAT: a fast spliced aligner with low memory  requirements." Nature methods 12.4 (2015): 357. |
| lncRNA-seq | stringtie   | 转录本组装            | v2.0       | -g 1 --no-coverage-search -r 50  --mate-std-dev 80           | Pertea M, Pertea GM, Antonescu CM, Chang TC, Mendell JT & Salzberg  SL. StringTie enables improved reconstruction of a transcriptome from RNA-seq  reads Nature Biotechnology 2015, doi:10.1038/nbt.3122 |
| lncRNA-seq | cpc         | 新转录本编码预测      | 0.9-r2     |                                                              | Kong, L., Zhang, Y., Ye, Z.-Q., Liu, X.-Q., Zhao, S.-Q., Wei, L., &  Gao, G. (2007). CPC: Assess the protein-coding potential of transcripts using  sequence features and support vector machine. Nucleic Acids Research, 35(Web  Server issue), W345-349. https://doi.org/10.1093/nar/gkm391 |
| lncRNA-seq | cnci        | 新转录本编码预测      | v2.0       |                                                              | Liang Sun, Haitao Luo, Dechao Bu, Guoguang Zhao, Kuntao Yu, Changhai  Zhang, Yuanning Liu, RunSheng Chen and Yi Zhao* Utilizing sequence intrinsic  composition to classify protein-coding and long non-coding transcripts.  Nucleic Acids Research (2013), doi: 10.1093/nar/gkt646 |
| lncRNA-seq | Deseq2      | 表达差异统计          | v3.12.1    |                                                              | Anders S, Huber W (2010). “Differential expression analysis for sequence  count data.” Genome Biology, 11, R106. doi: 10.1186/gb-2010-11-10-r106,  http://genomebiology.com/2010/11/10/R106/. |
| lncRNA-seq | INFERNAL    | lnc家族分析           | v1.1.2     | #NAME?                                                       | Nawrocki E P, Eddy S R. Infernal 1.1: 100-fold faster RNA homology  searches[J]. Bioinformatics, 2013, 29(22): 2933-2935. |
| lncRNA-seq | RNAplex     | antisense分析         | v0.2       |                                                              | Tafer H, Hofacker I L. RNAplex: a fast tool for RNA–RNA interaction  search[J]. Bioinformatics, 2008, 24(22): 2657-2663. |
| lncRNA-seq | R\prcomp    | 主成分分析            | v3.5.2     |                                                              | R Core Team (2018). R: A language and environment for statistical  computing. R Foundation for Statistical Computing, Vienna, Austria. URL  https://www.R-project.org/. |
| lncRNA-seq | rMATS       | 可变剪接分析          | v4.0.1     |                                                              | http://rnaseq-mats.sourceforge.net/index.html                |
| lncRNA-seq | GATK        | call SNP/indel        | 3.4-46     |                                                              | Van der Auwera GA, et al. From FastQ Data to High-Confidence Variant  Calls: The Genome Analysis Toolkit Best Practices Pipeline. CURRENT PROTOCOLS  IN BIOINFORMATICS 2013 43:11.10.1-11.10.33 |
| lncRNA-seq | ANNOVAR     | 变异分析              | 2016/2/1   | -window 35 -cluster 3 -filterName FS -filter "FS > 30.0"  -filterName QD -filter "QD < 2.0" | Wang K, Li M, Hakonarson H. ANNOVAR: Functional annotation of genetic  variants from next-generation sequencing data Nucleic Acids Research,  38:e164, 2010 |
| lncRNA-seq | BLAST+      | 比对                  | 2.8.0+     | -evalue 1e-5                                                 |                                                              |
| lncRNA-seq | miRPara     | miRNA前体预测         | v6.3       |                                                              | Wu Y, Wei B, Liu H, Li T, Rayner S. MiRPara: a SVM-based software tool  for prediction of most probable microRNA coding regions in genome scale  sequences. BMC Bioinformatics. 2011;12:107.  doi:10.1186/1471-2105-12-107. |


## Genome-sequence-workflow-parameter

| 版本号       | 数据库/软件 | 功能                                                     | 版本                  | 参数                                                         | 参考文献/数据库地址                                          |
| ------------ | ----------- | -------------------------------------------------------- | --------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
| 重测序       | bwa         | 参考基因组比对                                           | 0.7.15                | mem -k 32 -M                                                 | Fast and accurate short read  alignment with Burrows–Wheeler transform \| Bioinformatics \| Oxford Academic.  (n.d.). Retrieved 18 February 2020, from  https://academic.oup.com/bioinformatics/article/25/14/1754/225615 |
| 重测序       | samblaster  | 序列标记                                                 | 1.129                 |                                                              | Faust G G, Hall I M. SAMBLASTER:  fast duplicate marking and structural variant read extraction[J].  Bioinformatics, 2014, 30(17): 2503-2505. |
| 重测序       | bedtools    | 覆盖度统计                                               | v2.25.0               |                                                              | http://bedtools.readthedocs.io/en/latest/                    |
| 重测序       | manta       | SV检测                                                   | 1.1.1                 |                                                              | Chen X, Schulz-Trieglaff O, Shaw R,  et al. Manta: rapid detection of structural variants and indels for germline  and cancer sequencing applications[J]. Bioinformatics, 2015, 32(8):  1220-1222. |
| 重测序       | cnvnator    | CNV检测                                                  | 0.3.2                 | e-value < 0.01,Deletion区域的RD < 0.7；Duplication区域的RD >  1.3,CNV区间大小 > 1kb | Abyzov A, Urban A E, Snyder M, et  al. CNVnator: An approach to discover, genotype, and characterize typical and  atypical CNVs from family and population genome sequencing[J]. Genome  Research, 2011, 21(6):974. |
| 重测序       | GATK        | 变异检测                                                 | 3.4-46                | UnifiedGenotyper -glm BOTH                                   | Van der Auwera GA, et al. From FastQ  Data to High-Confidence Variant Calls: The Genome Analysis Toolkit Best  Practices Pipeline. CURRENT PROTOCOLS IN BIOINFORMATICS 2013  43:11.10.1-11.10.33 |
| 重测序       | ANNOVAR     | 变异注释                                                 | 2016/2/1              |                                                              | Wang K, Li M, Hakonarson H. ANNOVAR:  functional annotation of genetic variants from high-throughput sequencing  data[J]. Nucleic Acids Research, 2010, 38(16):e164. |
| 重测序       | treebest    | 进化树构建                                               | 1.9.2 build 25Dec2007 | neighbor-joining methods                                     | http://treesoft.sourceforge.net/treebest.shtml               |
| 重测序       | plink       | 数据格式转换                                             | v1.9                  |                                                              | Chang C C, Chow C C, Tellier L C A  M, et al. Second-generation PLINK: rising to the challenge of larger and  richer datasets[J]. Gigascience, 2015, 4(1): 7. |
| 重测序       | gcta        | 主成成分分析                                             |                       | Yang J, Lee S H, Goddard M E, et al.  GCTA: a tool for genome-wide complex trait analysis[J]. The American Journal  of Human Genetics, 2011, 88(1): 76-82. |                                                              |
| 重测序       | admixture   | 群体结构分析                                             | 1.3.0                 |                                                              | D.H. Alexander, J. Novembre, and K.  Lange. Fast model-based estimation of ancestry in unrelated individuals.  Genome Research, 19:1655–1664, 2009. |
| GWAS         | GATK        | 变异检测                                                 | 3.4-46                | -Window 4，-filter "QD < 4.0 \|\| FS > 60.0 \|\| MQ < 40.0  "，-G_filter "GQ < 20" | Van der Auwera GA, et al. From FastQ  Data to High-Confidence Variant Calls: The Genome Analysis Toolkit Best  Practices Pipeline. CURRENT PROTOCOLS IN BIOINFORMATICS 2013  43:11.10.1-11.10.33 |
| GWAS         | ANNOVAR     | 变异注释                                                 | 2016/2/1              |                                                              | Wang K, Li M, Hakonarson H. ANNOVAR:  functional annotation of genetic variants from high-throughput sequencing  data[J]. Nucleic Acids Research, 2010, 38(16):e164. |
| GWAS         | treebest    | SNP进化树构建                                            | 1.9.2 build           |                                                              | EnsemblCompara GeneTrees: Complete,  duplication-aware phylogenetic trees in vertebrates. (n.d.). Retrieved 18  February 2020, from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2652215/ |
| GWAS         | plink       | 数据筛选与格式转换                                       | v1.9                  |                                                              | Chang C C, Chow C C, Tellier L C A  M, et al. Second-generation PLINK: rising to the challenge of larger and  richer datasets[J]. Gigascience, 2015, 4(1): 7. |
| GWAS         | gcta        | 主成成分分析                                             | version 1.25.2        |                                                              | Yang J, Lee S H, Goddard M E, et al.  GCTA: a tool for genome-wide complex trait analysis[J]. The American Journal  of Human Genetics, 2011, 88(1): 76-82. |
| GWAS         | admixture   | 群体结构分析                                             | 1.3.0                 |                                                              | D.H. Alexander, J. Novembre, and K.  Lange. Fast model-based estimation of ancestry in unrelated individuals.  Genome Research, 19:1655–1664, 2009. |
| GWAS         | SPAGeDi     | 群体亲缘关系分析                                         | v1.4c                 |                                                              | Hardy O J, Vekemans X. SPAGeDi: a  versatile computer program to analyse spatial genetic structure at the  individual or population levels[J]. Molecular ecology notes, 2002, 2(4):  618-620. |
| GWAS         | Haploview   | 连锁不平衡分析                                           | 4.2                   | ：-maxdistance 10000 -minMAF 0.05                            | Barrett J C, Fry B, Maller J, et al.  Haploview: analysis and visualization of LD and haplotype maps[J].  Bioinformatics, 2005, 21(2):263-5. |
| GWAS         | Tassel      | 全基因组关联分析                                         | v3                    |                                                              | Bradbury P J, Zhang Z, Kroon D E, et  al. TASSEL : software, for association, mapping, of complex, traits, in  diverse, samples. [J]. Bioinformatics, 2007, 23(19):2633-2635. |
| 选择压力分析 | popgenome   | Fst、π值、Tajima's D、Fu & Li's D*、Fu & Li's F*值等计算 | 2.2.4                 |                                                              | Pfeifer B, Wittelsbürger U,  Ramos-Onsins S E, et al. PopGenome: an efficient Swiss army knife for  population genomic analyses in R[J]. Molecular biology and evolution, 2014,  31(7): 1929-1936. |
| 可变剪切     | Whippet     | 可变剪切位点分析                                         | v0.11.1               |                                                              | Sterne-Weiler T, Weatheritt R J,  Best A J, et al. Efficient and accurate quantitative profiling of alternative  splicing patterns of any complexity on a laptop[J]. Molecular cell, 2018,  72(1): 187-200. e6. |
| eGWAS        | R\qqnorm    | 数据正态化                                               | v3.5.2                |                                                              | R Core Team (2018). R: A language  and environment for statistical computing. R Foundation for Statistical  Computing, Vienna, Austria. URL https://www.R-project.org/. |
| eGWAS        | peer        | 效应协变量探究                                           | v1.3                  |                                                              | Stegle, O., Parts, L., Piipari, M.,  Winn, J., & Durbin, R. (2012). Using probabilistic estimation of  expression residuals (PEER) to obtain increased power and interpretability of  gene expression analyses. Nature protocols, 7(3), 500. |
| eGWAS        | gcta        | PCA分析                                                  | version 1.25.2        |                                                              | Yang J, Lee S H, Goddard M E, et al.  GCTA: a tool for genome-wide complex trait analysis[J]. The American Journal  of Human Genetics, 2011, 88(1): 76-82. |
| eGWAS        | MatrixEQTL  | eGWAS分析                                                | Version 2.1.0         | Shabalin, A. A. (2012). Matrix eQTL:  ultra fast eQTL analysis via large matrix operations. Bioinformatics, 28(10),  1353-1358. |                                                              |
